script.js

```diff
@@ -1,12 +1,76 @@
 //You can edit ALL of the code here
+const allEpisodes = getAllEpisodes();
+const rootElem = document.getElementById("root");
+let select = document.getElementById("selectBox");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// creat cards
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  const mainElm = document.createElement("div");
+  //Addind a display functionality
+  const display = document.getElementById("display");
+  display.innerText = `Displaying ${episodeList.length}/${allEpisodes.length}episodes`;
+  mainElm.id = "main";
+  rootElem.appendChild(mainElm);
+  // iterate through each episode of the array
+  for (let i = 0; i < episodeList.length; i++) {
```

Comment:
you may change this loop with forEach. your all js codes looks pretty and this part draws attention